cookiecutter.json

```diff
@@ -17,5 +17,16 @@
     "has_spring_boot_session": "n",
     "java_version": "1.8",
     "spring_boot_authentication": ["basic", "jwt", "ldap", "oauth2"],
-    "spring_boot_version": "2.1.9.RELEASE"
+    "spring_boot_version": "2.1.9.RELEASE",
+  	"xhr_library": ["fetch", "axios", "superagent", "request", "none"], 
```

Comment:
Looks like this line (xhr_library) is still tab-indented. Could you please make it space-indented?